src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,173 @@
+     */
+    public String delete(Task delTask, int totalTask) {
+        return "Noted. I've removed this task: \n" + delTask
+                + "\nNow you have " + totalTask + " tasks in the list.";
+    }
+
+    /**
+     * Prints a message to notify the user that a task is completed.
+     *
+     * @param doneTask Task that is completed
+     */
+    public String done(Task doneTask) {
+        return "Nice! I've marked this task as done: \n" + doneTask;
+    }
+
+    /**
+     * Prints a message to notify the user that a task's status has been changed to <code>incomplete</code>.
+     *
+     * @param doneTask Task that is completed
+     */
+    public String undone(Task doneTask) {
+        return "The task's status has been changed to incomplete: \n" + doneTask;
+    }
+
+    /**
+     * Prints a message to notify the user that a new task has been added to the list.
+     *
+     * @param newTask Task to be added.
+     * @param totalTask Total number of tasks in the tasks list.
+     */
+    public String addTask(Task newTask, int totalTask) {
+        return "Got it. I've added this task: \n" + newTask
+                + "\nNow you have " + totalTask + " tasks in the list.";
+    }
+
+    /**
+     * Prints a message to notify the user that a task is already completed.
+     *
+     * @param doneTask Task that is already completed
+     */
+    public String showAlreadyDone(Task doneTask) {
+        return "You have already completed the task: \n" + doneTask;
+    }
+
+    /**
+     * Prints a message to warn the user when the system is unable to recognise the command.
+     */
+    public String showInvalidCommand() {
+        return "OOPS!!! I'm sorry, but I don't know what that means :-(";
+    }
+
+    /**
+     * Prints a message to warn the user when the index input for a <code>delete</code> or <code>done</code>
+     * command is invalid.
+     */
+    public String showInvalidTaskIndex() {
+        return "OOPS!!! The task is unavailable.";
+    }
+
+    /**
+     * Prints a message to warn the user when the description for a deadline is invalid.
+     */
+    public String showInvalidDeadlineDesc() {
+        return "OOPS!!! Wrong description format for deadline.\n\n"
+                + "Please enter your deadline in the format of: \n"
+                + "  \"deadline (description) /by (date & time)\"\n\n"
+                + "The date and time should be in the format of: \n"
+                + "  \"[yyyy-MM-dd HHmm]\"";
+    }
+
+    /**
+     * Prints a message to warn the user when the description for a event is invalid.
+     */
+    public String showInvalidEventDesc() {
+        return "OOPS!!! Wrong description format for event.\n"
+                + "  \"event (description) /at (date & time)\"\n\n"
+                + "The date and time should be in the format of: \n"
+                + "  \"[yyyy-MM-dd HHmm]\"";
+    }
+
+    /**
+     * Prints a message to warn the user when the description for a todo is empty.
```

Comment:
Hmm, I like how every function has a Javadoc description of what each function does. 😁
However, it seems a bit unnecessary to create every function for every error message.
(Just an opinion)